autopush/web/webpush.py

```diff
@@ -382,18 +385,25 @@ def _router_completed(self, response, uaid_data, warning=""):
             d = deferToThread(self.ap_settings.router.register_user,
                               uaid_data)
             response.router_data = None
-            d.addCallback(lambda x: self._router_completed(response,
-                                                           uaid_data,
-                                                           warning))
+            d.addCallback(lambda x: self._router_completed(
+                response,
+                uaid_data,
+                warning,
+                notification))
             return d
         else:
             # No changes are requested by the bridge system, proceed as normal
             if response.status_code == 200 or response.logged_status == 200:
                 self.log.info(format="Successful delivery",
                               client_info=self._client_info)
             elif response.status_code == 202 or response.logged_status == 202:
-                self.log.info(format="Router miss, message stored.",
-                              client_info=self._client_info)
+                self.log.info(
+                    format="Router miss, message stored.",
+                    message_size=len(notification.data) if notification else 0,
```

Comment:
This was _client_info['message_size'] in endpoint. let's just dump the notification metadata in there up front so we don't have to the notification around